python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -245,7 +245,7 @@ def test_upscale_downscale_delay():
     downscale_delay_s = 600.0
 
     config = AutoscalingConfig(
-        min_replicas=1,
+        min_replicas=0,
```

Comment:
Same comment as [above](https://github.com/ray-project/ray/pull/24892/files#r883913049).

Reply:
this for testing multiple scenarios. it is already covered existing replicas to scale and 0 replicas to scale up